tests/unit/dojson/test_dojson_hepnames.py

```diff
@@ -174,6 +174,82 @@ def test_ids_from_035__a_9_with_malformed_cern():
     assert expected == result['ids']
 
 
+def test_ids_from_035__a_9_with_cern_lowercase():
+    snippet = (
+        '<datafield tag="035" ind1=" " ind2=" ">'
+        '  <subfield code="9">CERN</subfield>'
+        '  <subfield code="a">cern-783683</subfield>'
+        '</datafield>'
+    )  # record/1408145/export/xme
+
+    expected = [
+        {
+            'type': 'CERN',
+            'value': 'CERN-783683',
+        },
+    ]
+    result = clean_record(hepnames.do(create_record(snippet)))
+
+    assert expected == result['ids']
+
+
+def test_ids_from_035__a_9_with_cerm():
+    snippet = (
+        '<datafield tag="035" ind1=" " ind2=" ">'
+        '  <subfield code="9">CERN</subfield>'
+        '  <subfield code="a">CERM-724319</subfield>'
+        '</datafield>'
+    )  # record/1244430/export/xme
+
+    expected = [
+        {
+            'type': 'CERN',
+            'value': 'CERN-724319',
+        },
+    ]
+    result = clean_record(hepnames.do(create_record(snippet)))
+
+    assert expected == result['ids']
+
+
+def test_ids_from_035__a_9_with_cner():
```

Comment:
Yeah, when I write code I usually add a test at a time, which is why I end up with so many. I will refactor these into one.